src/_utils/colorHelper.ts

```diff
@@ -0,0 +1,9 @@
+import { theme, ThemeColor } from '../_styles/theme';
+
+export function getColor(input: ThemeColor) {
```

Comment:
getColorTypeFromString?

Reply:
`getColorFromString` zou kunnen, maar ik vind dit leesbaarder en het is getyped dus op zich kan er weinig misgaan imo